examples/trader.js

```diff
@@ -7,19 +7,20 @@
  * You can ask the bot to trade with a villager, display the villagers in range
  * and show what trades a villager has by sending a chat message.
  */
-const mineflayer = require('mineflayer')
+import { createBot } from 'mineflayer'
 
 if (process.argv.length < 4 || process.argv.length > 6) {
   console.log('Usage : node trader.js <host> <port> [<name>] [<password>]')
   process.exit(1)
 }
+
 console.log('Commands :\n' +
   '  show villagers\n' +
   '  show inventory\n' +
   '  show trades <id>\n' +
   '  trade <id> <trade> [<times>]')
```

Comment:
Unneeded in my opinion as the file should act as example, not a polished product. @rom1504, @Karang what do you think? I'm tagging you both because it's been here before the PR